torch/_inductor/scheduler.py

```diff
@@ -1477,7 +1477,15 @@ def __init__(self, nodes: List[ir.Operation]) -> None:
         self.create_foreach_nodes()
         self.topological_sort_schedule()
         self.logged_slow_fusion: Set[Tuple[str, str]] = set()
+        self.nodes = spmd_bucketing.bucketing_per_blcok(self.nodes)
```

Comment:
let's remove this as it is not relevant to this PR -- let's address one problem at a time.